waltz-model/src/main/java/com/khartec/waltz/model/survey/SurveyQuestion.java

```diff
@@ -0,0 +1,24 @@
+package com.khartec.waltz.model.survey;
+
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.khartec.waltz.model.IdProvider;
+import org.immutables.value.Value;
+
+import java.util.Optional;
+
+@Value.Immutable
+@JsonSerialize(as = ImmutableSurveyQuestion.class)
+@JsonDeserialize(as = ImmutableSurveyQuestion.class)
+public abstract class SurveyQuestion implements IdProvider {
+
+    public abstract Long surveyTemplateId();
+    public abstract String questionText();
+    public abstract Optional<String> helpText();
```

Comment:
think it will cause problems with the Mapper as it currently stands as the immutable will reject.  I think you need an `Optoinal.ofNullable(record.getHelpText())`